torch/csrc/utils/python_scalars.h

```diff
@@ -95,8 +95,12 @@ inline void store_scalar(void* data, at::ScalarType scalarType, PyObject* obj) {
       *(at::Float8_e4m3fnuz*)data =
           at::convert<at::Float8_e4m3fnuz, double>(THPUtils_unpackDouble(obj));
       break;
+    case at::kFloat8_e8m0fnu:
```

Comment:
Make this macro based here and below?

Reply:
tracking in https://github.com/pytorch/pytorch/issues/146647